server/service/service_users.go

```diff
@@ -29,16 +29,11 @@ func (svc *Service) CreateInitialUser(ctx context.Context, p fleet.UserPayload)
 }
 
 func (svc *Service) newUser(ctx context.Context, p fleet.UserPayload) (*fleet.User, error) {
-	var ssoEnabled bool
-	if (p.SSOInvite != nil && *p.SSOInvite) || (p.SSOEnabled != nil && *p.SSOEnabled) {
-		ssoEnabled = true
-	}
```

Comment:
I don't think we want this removed. (`Service.newUser` is used by `Service.CreateUserFromInvite`, `Service.CreateUser` and `Service.CreateInitialUser`)

Reply:
I wonder why no test failed...